earth_enterprise/src/fusion/autoingest/StorageManager.h

```diff
@@ -188,9 +241,9 @@ StorageManager<AssetType>::Get(
   return entry;
 }
 
-template<class AssetType>
-typename StorageManager<AssetType>::PointerType
-StorageManager<AssetType>::GetEntryFromCacheOrDisk(const AssetKey & ref) {
+template<class AssetType, class AssetStorageType>
```

Comment:
Should update this function to use the serializer as well instead of calling Load directly.

Reply:
I think those new calls came in after I merged in from master, and I just didn't notice it. Nice catch.